liquid.py

```diff
@@ -237,17 +260,60 @@ def alloc(job, group):
     print(first(running, 'running allocations'))
 
 
-def nomad_address():
+def collection_exists(name):
+    """Returns true if the collection with the given name was declared in the ini file."""
+
+    names = dict(config.collections).keys()
+    return reduce(lambda exists, curr_name: exists or curr_name.lower() == name.lower(), names, False)
+
+
+def initcollection(name):
+    """Initialize collection with given name.
+    
+    Create the snoop database, create the search index, run dispatcher, add collection
+    to search.
+    
+    :param name: the collection name
+    """
+    if not collection_exists(name):
+        raise RuntimeError('Collection %s does not exist in the liquid.ini file.', name)
+
+    container = f'snoop-{name}-api'
+
+    try:
+        shell_cmd(container, './manage.py', 'initcollection')
+    except CalledProcessError as e:
+        if not re.search(f'index \[{name}/[^\]]+] already exists', e.output.decode()):
+            log.error(e.output.decode())
+            raise
+        log.info(f'Index {name} already exists.')
+
+    try:
+        snoop_url = first(nomad.agent_members())['Addr'] + f':8765/{name}/collection/json'
+        shell_cmd('hoover-search', './manage.py', 'addcollection', name, '--index',
+                  name.lower(), snoop_url, '--public')
+    except CalledProcessError as e:
+        if not re.search('IntegrityError: duplicate keyy', e.output.decode()):
+            log.error(e.output.decode())
+            raise
+        log.info(f'Collection {name} already added to hoover search.')
+
+
+def get_nomad_address():
     """
     Print the nomad server's address.
     """
     members = [m['Addr'] for m in nomad.agent_members()]
-    print(first(members, 'members'))
+    return first(members, 'members')
+
+
+def nomad_address():
+    print(get_nomad_address())
 
 
 def set_collection_defaults(name, settings):
     settings['name'] = name
-    settings.setdefault('workers', '1')
+    settings.setdefault('workers', config.default_workers_no)
```

Comment:
I don't think `'1'` is worth extracting as a default.

Reply:
I'd rather have it in the config

Reply:
For sure the workers number should be in the config. But the default can be hardcoded right here, as it was, I think the code was much easier to read.

Reply:
Please don't change the `'1'` here.